behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
```

Comment:
useless. The composer autoloader is already loaded automatically by Behat. Adding a bootstrap file is useful when a *custom* file needs to be loaded